### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by stefano-ottolenghi**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-dataflow-connector-21.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-dataflow-connector-21.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-dataflow-connector-21.surge.sh

File: common-content-dataflow/modules/ROOT/partials/run-job.adoc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 = Run an import job
 
+[TIP]
+You can validate your specification file locally via the link:https://github.com/neo4j-contrib/local-dataflow-runner/[Local Dataflow Runner].
+The tool allows you to verify that your job file conforms to the template specifications, but doesn't attempt to run it (so there may still be runtime errors). This form of validation is much quicker than submitting a Dataflow job.
```
Comment (fbiville) [inline common-content-dataflow/modules/ROOT/partials/run-job.adoc:5]:
```suggestion
The tool allows you to locally verify that your job file conforms to the template specification and execute it against a temporary Neo4j container. It relies on [Apache Beam's Direct Runner](https://beam.apache.org/documentation/runners/direct/) (there may still be runtime errors in Dataflow). This form of validation is much quicker than submitting a Dataflow job.
```

**Review by fbiville — APPROVED** [review]

**Pull request merged**